vtex/faststore
# feat(cli): add support for the `init` command

## What's the purpose of this pull request?

Adds initial support for the `init` command, which aims to boostrap (and scaffold) Discovery from the starter.store template. 

## How it works?

It creates a new `oclif` command that wraps the discovery composition using `degit`. 

## How to test it?

- Clone https://github.com/vtex/store-monorepo
- Recommended: checkout to the `faststore-cli-integration` branch
- On `package.json` change the build version of `@faststore/cli` to the last one available in our CodeSandbox CI
- Run `yarn` again to refresh the modules
- Run `yarn fsp discovery init`
- Notice that `discovery/` will be created and available for usage.
